CodeToReview.java

```diff
@@ -0,0 +1,250 @@
+                case 7:
+                    if (a7 == "7") {
+                    a7 = "O";
+                    }
+                    break;
+                case 8:
+                    if (a8 == "8") {
+                    a8 = "O";                        
+                    }
+                    break;
+                case 9:
+                    if (a9 == "9") {
+                    a9 = "O";
+                    }
+                    break;
+                default:
+                    System.out.printf("An error occured. Please accept it politely and restart the game.");
+                    break;
+            }
+            System.out.printf("\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n");
+            if (null != Winners(a1,a2,a3,a4,a5,a6,a7,a8,a9))switch (Winners(a1,a2,a3,a4,a5,a6,a7,a8,a9)) {
+                case "NONE":
+                    if (a1 != "1" && a2 != "2" && a3 != "3" && a4 != "4" && a5 != "5" && a6 != "6" && a7 != "7" && a8 != "8" && a9 != "9") {
+                        winner = true;
+                        System.out.println("DRAW!");
+                        Thread.sleep(3000);
+                        break;
+                    } else {
+                    PrintBoard(a1,a2,a3,a4,a5,a6,a7,a8,a9);
+                    System.out.printf("Player X's turn: ");
+                    inX = input.nextInt();
+                    break;
+                    }
+                case "X":
+                    winner = true;
+                    System.out.println("X HAS WON!!!");
+                    Thread.sleep(3000);
+                    break;
+                case "O":
+                    winner = true;
+                    System.out.printf("O HAS WON!!!");
+                    Thread.sleep(3000);
+                    break;
+                default:
+                    break;
+            }
+            if (winner) {
+                break;
+            }
+            switch (inX) {
+                case 1:
+                    if (a1 == "1") {
+                    a1 = "X";
+                    }
+                    break;
+                case 2:
+                    if (a2 == "2") {
+                    a2 = "X";
+                    }
+                    break;
+                case 3:
+                    if (a3 == "3") {
+                    a3 = "X";
+                    }
+                    break;
+                case 4:
+                    if (a4 == "4") {
+                    a4 = "X";
+                    }
+                    break;
+                case 5:
+                    if (a5 == "5") {
+                    a5 = "X";
+                    }
+                    break;
+                case 6:
+                    if (a6 == "6") {
+                    a6 = "X";
+                    }
+                    break;
+                case 7:
+                    if (a7 == "7") {
+                    a7 = "X";
```

Comment:
Might be worth putting "X" static value in a constant and reference the constant in the code.
Makes it easier to read, and if the static value was to change, it would make the code changes considerably easier.